src/utils/db.js

```
@@ -0,0 +1,24 @@
+const { Sequelize } = require('sequelize');
+
+require('dotenv').config();
+
+const client = new Sequelize(
+  process.env.PGDATABASE,
+  process.env.PGUSER,
+  process.env.PGPASSWORD,
```

Sequelize constructor is used with positional args which is valid, but ensure the environment variables are present. If any of `process.env.PGDATABASE`, `process.env.PGUSER`, or `process.env.PGPASSWORD` are missing, the connection will fail with a possibly unhelpful error. Consider validating required env vars (and failing fast with a clear message) or using a connection URI (process.env.DATABASE_URL) for clarity.